src/main/src/main/java/org/geoserver/catalog/impl/WMSLayerInfoImpl.java

```diff
@@ -125,7 +126,7 @@ public String getPrefferedFormat() {
 
     @Override
     public void setPrefferedFormat(String prefferedFormat) {
-        this.prefferedFormat = prefferedFormat;
+        this.prefferedFormat = (prefferedFormat == null) ? "image/png" : prefferedFormat;
```

Comment:
"preferred" here too.

Reply:
there is a high chance, this spelling change in methods will break the XML persist/de-persist, as in might cause data directory issues.

anyway will correct the names

Reply:
You are indeed right, the old xml entry will be ignored. However, this is a recent change that landed on master only, right? So we are not breaking any backwards compat with released software.

Reply:
will make the change